cpp/test/neighbors/ann_cagra_c.cu

```diff
@@ -34,9 +34,14 @@ float queries[4][2] = {{0.48216683, 0.0428398},
                        {0.51260436, 0.2643005},
                        {0.05198065, 0.5789965}};
 
+int64_t filter[2] = {1, 2};
```

Comment:
Adapt this test input in consequence.

Reply:
I've adapted the `_search` function and the test so that it doesn't create a new bitset